### PR TITLE
Add “Ask for Help”

### PR DESCRIPTION
**Pull request by jonathansimmons**

### CODE REVIEW

File: appendix-11-asking for help.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+title: "Asking for help"
+chapter: "Chapter A10:"
+prev_link: "← Holidays"
+prev_url: "/appendix-09-holidays"
+---
+
+As a fully distributed team, remote calls our lifeblood for our communication. Simple joining a remote team doesn't make you great at communicating remotely, though, and at some point, we're all going to need help. So who do we turn to? How do we ask? Here are a few essential tips we've developed to successfully ask for help within a remote team.
```
Comment (davidlink) [inline appendix-11-asking for help.md]: > remote calls our lifeblood

should this be "remote calls are our lifeblood"?

> Simple joining a remote team

should this be "Simply joining"?

File: appendix-11-asking for help.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## Be willing to ask
+We all need help, and for it's inherently human to feel embarrassed to ask for it. This is something as a team we strive to push through. Asking for help is strength, not weakness. Asking for help grows you, and it empowers the individual who knows the answer to practice sharing that information. This is a balance, of course. There is such a thing as too many questions in a given period. Do your research, If it's a work question, check the handbook. A code question, stack overflow. Use your discretion, and when all else fails, apply the 5-minute rule and phone a friend.
```
Comment (davidlink) [inline appendix-11-asking for help.md]: > We all need help, and for it's inherently human

We all need help, and ~~for~~ it's inherently human

> Do your research, If it's a work

I think maybe that should be a period.

File: appendix-11-asking for help.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+## Requestee, please communicate your situation
+As much as it's essential for the person asking to communicate their needs, it's equally vital that you help them understand your situation. If the requester has done their job of "communicating their need," which is not always the case, success now depends on you responding in kind by communicating your situation or maybe even asking them to express their need more clearly.
+
+That said, just because someone has asked for your time doesn't mean you're in a ready state and position to give it. The expectation is that you communicate your situation by:
+
+### Ask clarifying questions
+Needs are not always clearly communicated. It's now your job to help them understand what you don't. Maybe this means asking for more info about the problem or asking clarifying questions like, "do you need to show me something, or can this just be a phone call?".
+
+### Communicating your obstacles
+As we said, you might not be free, and it's ok to communicate that. "I'm in the zone and won't be out available 1pm", or "I've got my child with me for the next hour" or even "I can talk, but my kids are in the room so it might be a bit loud."
```
Comment (davidlink) [inline appendix-11-asking for help.md]: > I'm in the zone and won't be out available 1pm

"I'm in the zone and won't be available until 1pm"?

File: appendix-11-asking for help.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+Doing these things allows the requestor to understand your situation and decide how to proceed.
```
Comment (davidlink) [inline appendix-11-asking for help.md]: > requestor

"requester" for consistency.

**Review by blakeprudhomme — APPROVED** [review]
This looks good. I made a few small changes. Just noting that I did not see this page linked to from anywhere.

File: appendix-11-asking for help.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+While these may seem simple, and they can be, they are often quickly forgotten when asking someone for help. Practicing our ability to communicating our needs and situation will make us a more reliable remote team, who respect our teammates and optimize our time spent collaborating.
```
Comment (davidlink) [inline appendix-11-asking for help.md]: > Practicing our ability to communicating

"Practicing our ability to communicate"

File: appendix-11-asking for help.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+## Be understanding
+But what if the person I'm asking for help doesn't respond? This is a real possibility; maybe they are away from their phone or just in the zone. Their lack of response might feel like they aren't communicating. However, if we step out of our own selfish need for a second, we'll see that they are communicating. Their silence is telling us that they're not available. We need to be able to respect that.
+
+Does the practice "being understanding" give a coworker carte blanche to ignore your request for help for weeks, no. It is, however, essential for us to understand that a primary goal of being a remote team is to create space for everyone to focus. So if they don't respond, they're probably just busy.
```
Comment (davidlink) [inline appendix-11-asking for help.md]: > Does the practice "being understanding"

'Does the practice of "being understanding"'

> ignore your request for help for weeks, no.

"ignore your request for help for weeks? No."

**Review by davidlink — DISMISSED** [review]

File: appendix-11-asking for help.md (hunk 1 above)
Comment (blakeprudhomme) [reply appendix-11-asking for help.md]: I fixed these.

File: appendix-11-asking for help.md (hunk 2 above)
Comment (blakeprudhomme) [reply appendix-11-asking for help.md]: These are both addressed!

**Pull request merged**